googleapis-common-protos-types/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Release History
+
+### 1.0.4 / April 3 2019
```

Comment:
I think a `yyyy-mm-dd` date format might play better with releasetool, just in case it is introduced for this project.

Reply:
Sounds good. I've updated the date format.